x-pack/plugins/security_solution/server/endpoint/services/actions/clients/crowdstrike/crowdstrike_actions_client.test.ts

```diff
@@ -75,6 +79,70 @@ describe('CrowdstrikeActionsClient class', () => {
     });
   });
 
+  it('should save response with error in case of actionResponse containing errors', async () => {
+    const doc = {
```

Comment:
can you add types to this hard coded variables? or use the generator, which already returned properly typed data.